cordova-lib/spec-cordova/emulate.spec.js

```diff
@@ -82,7 +85,7 @@ describe('emulate command', function() {
             cordova.raw.emulate({platforms: ['ios'], options: {optionTastic: true }}).then(function(err) {
                 expect(prepare_spy).toHaveBeenCalledWith(jasmine.objectContaining({platforms: ['ios']}));
                 expect(getPlatformApi).toHaveBeenCalledWith('ios');
-                expect(platformApi.run).toHaveBeenCalledWith({ device: false, emulator: true, optionTastic: true });
+                expect(platformApi.run).toHaveBeenCalledWith({ device: false, emulator: true, optionTastic: true, nobuild: true });
```

Comment:
@daserge, could you please add another expectation to check if `build` also had been called and changing the options in `build` does not affect `run` command?